src/main/java/base/BaseTest.java

```diff
@@ -1,23 +1,18 @@
 package base;
 
-import config.TestProperties;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
-import page.HomePage;
+import PageFactory.HomePageFactory;
 
 import static driver.Driver.getDriver;
 import static driver.Driver.tearDown;
 
 public class BaseTest {
 
-    protected static HomePage homePage;
+    protected static HomePageFactory homePageFactory;
 
```

Comment:
let's include method logIn here, using combinations of methods calls for the already defined page objects